src/main/scala/centaur/test/formulas/TestFormulas.scala

```diff
@@ -64,4 +65,22 @@ object TestFormulas {
       _ <- validateDirectoryContentsCounts(workflowDefinition, testWf)
     } yield ()
   }
+  
+  private def cromwellRestart(workflowDefinition: Workflow, testResume: Boolean): Test[Unit] = CentaurConfig.runMode match {
+    case ManagedCromwellServer(_, postRestart, withRestart) if withRestart =>
+      for {
+        w <- submitWorkflow(workflowDefinition)
+        jobId <- pollUntilCallIsRunning(w, "cromwell_restart.cromwell_killer")
```

Comment:
This seems pretty magical, can it be a knob that explicitly calls it out as the Cromwell Killer?